app/models/setting/provisioning.rb

```diff
@@ -52,6 +52,7 @@ def self.default_settings
       self.set('query_local_nameservers', N_("Foreman will query the locally configured resolver instead of the SOA/NS authorities"), false, N_('Query local nameservers')),
       self.set('remote_addr', N_("If Foreman is running behind Passenger or a remote load balancer, the IP should be set here. This is a regular expression, so it can support several load balancers, i.e: (10.0.0.1|127.0.0.1)"), "127.0.0.1", N_('Remote address')),
       self.set('token_duration', N_("Time in minutes installation tokens should be valid for, 0 to disable token generation"), 60 * 6, N_('Token duration')),
+      self.set('ssh_timeout', N_("Time in seconds before SSH provisioning times out"), 60 * 6, N_('SSH timeout')),
```

Comment:
I would add the explanation of the unit that is used here.
For example "microseconds" for better understanding the calculation, and the value to place when setting it up.

Reply:
I was assuming users would enter a time in seconds (Timeout default), which is why I made the help text say "Time in seconds ... ".  I tried to follow how `token_duration` does it.  Is there something more that you think I should add?

Reply:
Can you actually make the default value of 2 minutes? Our old default was way to big, browser will kick you out already. Since it's configurable now users can change this?

Reply:
Sure, I'll change it to 2 minutes.